solidity/ecdsa/tasks/request-new-wallet.ts

```diff
@@ -0,0 +1,31 @@
+import { task } from "hardhat/config"
+
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+
+task("request-new-wallet", "Requests for a new ECDSA wallet")
+  .addParam("walletOwnerAddress", "The Wallet Owner's address")
+  .setAction(async (args, hre) => {
+    const { walletOwnerAddress } = args
+
+    await requestNewWallet(hre, walletOwnerAddress)
+  })
+
+async function requestNewWallet(
+  hre: HardhatRuntimeEnvironment,
+  walletOwnerAddress: string
+) {
+  const { ethers, helpers } = hre
+
+  if (!ethers.utils.isAddress(walletOwnerAddress)) {
+    throw Error(`invalid address: ${walletOwnerAddress}`)
+  }
+
+  const walletOwner = await ethers.getSigner(walletOwnerAddress)
+
+  const walletRegistry = await helpers.contracts.getContract("WalletRegistry")
+
+  const tx = await walletRegistry.connect(walletOwner).requestNewWallet()
```

Comment:
I am not sure if this is correct. In the deployment scripts we transfer the wallet owner role to the `Bridge`, so calling the function from the `Bridge` address should not be possible.

Also, I think we no longer need this task after https://github.com/keep-network/keep-core/pull/3253 - we can call beacon genesis and request a new wallet from CLI.

What about closing this PR without merging?